File: cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int cents;
    int coins = 0;

    do
    {
        printf("Change owed : ");
        scanf("%d", &cents);
    }

    while (cents < 0);

    // Calculate quarters
    coins += cents / 25;
    cents %= 25;

    // Calculate dimes
    coins += cents / 10;
    cents %= 10;

    // Calculate nickels
    coins += cents / 5;
    cents %= 5;

    // Calculate pennies
    coins += cents;

    printf("%d\n", coins);

    return 0;
}

File: sentimental-cash/cash.py
from cs50 import get_float


def main():

    while True:
        change = get_float("Change owed: ")
        if change >= 0:
            break

    cents = round(change * 100)

    coins = 0

    coins += cents // 25  # Quarters
    cents %= 25

    coins += cents // 10  # Dimes
    cents %= 10

    coins += cents // 5   # Nickels
    cents %= 5

    coins += cents // 1   # Pennies

    print(coins)


if __name__ == "__main__":
    main()
